File: MyStagePass.Services/Database/Status.cs
namespace MyStagePass.Services.Database
{
	public class Status 
	{
		public const int PendingID = 1;
		public const int ApprovedID = 2;
		public const int RejectedID = 3;
		public int StatusID {  get; set; }
		public string StatusName {  get; set; }
		public ICollection<Event> Events { get; set; } = new List<Event>();
	}
}

File: UI/mystagepass_admin/lib/models/Status/status.dart
import 'package:json_annotation/json_annotation.dart';
import '../Event/event.dart';

part 'status.g.dart';

@JsonSerializable()
class Status {
  int? statusId;
  String? statusName;
  List<Event>? events;

  Status({this.statusId, this.statusName, this.events});

  factory Status.fromJson(Map<String, dynamic> json) => _$StatusFromJson(json);
  Map<String, dynamic> toJson() => _$StatusToJson(this);
}
